ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -209,7 +199,7 @@ private Context getContext() {
     if (mActivity != null) {
```

Comment:
I think this should be `return Assertions.assertNotNull(mActivity)` so we still crash if it is null.